packages/transformers/babel/src/config.js

```diff
@@ -111,16 +111,16 @@ export async function load(
       warnOldVersion();
 
       if (typeof partialConfig.babelrcPath === 'string') {
-        addIncludedFile(partialConfig.babelrcPath);
+        await addIncludedFile(partialConfig.babelrcPath);
```

Comment:
I remember suggesting that these awaits are unnecessary on the devdep PR, and then Devon removed them before merging that.